apollo-router/src/plugins/telemetry/config_new/instruments.rs

```diff
@@ -105,6 +103,25 @@ pub(crate) enum InstrumentType {
     /// A monotonic counter
     Counter,
 
-    /// A duration histogram
+    /// A counter
+    UpDownCounter,
+
+    /// A histogram
+    Histogram,
+
+    /// A counter
+    Gague,
```

Comment:
typo and bad doc

Reply:
[a120e6a](https://github.com/apollographql/router/pull/4061/commits/a120e6ae3b0e323f72eeab9db85092f55a15fd90)